airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/typing_deduping/PostgresSqlGenerator.java

```diff
@@ -60,8 +61,28 @@ public class PostgresSqlGenerator extends JdbcSqlGenerator {
       "timestamptz", "timestamp with time zone",
       "timetz", "time with time zone");
 
+  private final NamingConventionTransformer namingTransformer;
+
   public PostgresSqlGenerator(final NamingConventionTransformer namingTransformer) {
     super(namingTransformer);
+    this.namingTransformer = namingTransformer;
+  }
+
+  @Override
+  public StreamId buildStreamId(final String namespace, final String name, final String rawNamespaceOverride) {
+    // There is a mismatch between convention used in create table query in SqlOperations vs this.
+    // For postgres specifically, when a create table is issued without a quoted identifier, it will be
+    // converted to lowercase.
+    // To keep it consistent when querying raw table in T+D query, convert it to lowercase.
+    // TODO: This logic should be unified across Raw and final table operations in a single class
+    // operating on a StreamId.
+    return new StreamId(
+        namingTransformer.getNamespace(namespace),
+        namingTransformer.convertStreamName(name),
+        namingTransformer.getNamespace(rawNamespaceOverride),
```

Comment:
do we need to lowercase the schema name as well? since I assume it's also not quoted

Reply:
I guess, i'll do that for consistency if they provided a mixed case `raw_namespace_override`